src/App.jsx

```diff
@@ -1,33 +1,73 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react"
+import "./App.css"
+import Inbox from "./components/Inbox"
+import InboxDetails from "./components/InboxDetails"
+import NavBar from "./components/NavBar"
+import Search from "./components/Search"
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [emailData, setEmailData] = useState([])
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails")
+      .then((response) => response.json())
+      .then((data) => {
+        setEmailData(data.data)
+      })
+  }, [])
+
+  const [hiddenToggle, setHiddenToggle] = useState("hidden")
+
+  function handleClick() {
+    if (hiddenToggle === "hidden") {
```

Comment:
I would usually use a boolean for tracking a `true` or `false` state. 

I think these are Tailwind class names - you don't really want to make Tailwind's class names part of your app's logic, what if Tailwind updates and changes? Which parts of your code do you now need to change?